### ISSUE TITLE
logits = Lambda(lambda z: z/temperature)(x)

### ISSUE THREAD

**Issue by ElectricSquirrel** [chuckyee/cardiac-segmentation#5]
What's the use of this line of code?

dilatedunet.py
**logits = Lambda(lambda z: z/temperature)(x)**
![image](https://user-images.githubusercontent.com/9989714/31921966-d8d31cd2-b8a4-11e7-9ec3-abd0fd22d924.png)

**Comment by chuckyee** [comment]
When I had implemented just the u-net, the performance was lackluster. I thought perhaps forcing the model to be more "decisive" might help -- by decisive, I mean to force the model to mainly output 0 or 1 for each pixel-wise probability, rather than having many outputs near 0.5. This can be accomplished by rescaling the logits (by using an effective "temperature" must less than 1).

Ultimately, that wasn't the problem and using dilated convolutions helped.
